MatrixKit/Models/RoomList/MXKSessionRecentsDataSource.m

```diff
@@ -51,7 +51,15 @@ @interface MXKSessionRecentsDataSource ()
      */
     NSMutableDictionary<NSString*, NSArray<MXSpaceChildInfo *> *> *lastSuggestedRooms;
     
+    /**
+     Event listener of the current space used to update the UI if an event occurs.
+     */
     id spaceEventsListener;
+    
+    /**
+     Observer used to reload data when the space service is initialised
+     */
+    id spaceServiceDidInitialisedObserver;
```

Comment:
Very unimportant, but could be:
```suggestion
    id spaceServiceDidInitialiseObserver;
```
(Although I probably wouldn't worry seeing as it currently matches the SDK).